terraform/hcl/applicable/req_sw_terraform_aws_alb_https_only.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_lb_listener" "vulnerable_example" {
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 12 2 "vulnerable_example" /terraform/hcl/applicable/req_sw_terraform_aws_alb_https_only.tf)

## ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    high Infrastructure as Code Vulnerability 🛠️
	
Finding: AWS Load balancer using insecure communications

### 👇 Details

Resources `aws_lb_listener` and `aws_alb_listener` should set `protocol = "HTTPS"` (default is `"HTTP"`)

Vulnerable example -
```
resource "aws_lb_listener" "vulnerable_example" {
  protocol          = "HTTP"
}
```

Secure example -
```
resource "aws_lb_listener" "secure_example" {
  protocol          = "HTTPS"
}
```	

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)